src/init/pkg_installer.sh

```diff
@@ -263,9 +260,11 @@ else
 
     # Restore service
     if [ -n "${INIT_PATH}" ]; then
-        if [ $CHK_CONFIG -eq 1 ]; then
+        chk=$(which chkconfig &> /dev/null)
```

Comment:
Forwarding the output to `/dev/null` makes `$chk` empty even when `chkconfig` exists so the `if` will be always false.
```
root@wazuh:~# which cmake &> /dev/null
root@wazuh:~# which cmake
/usr/local/bin/cmake
```
```suggestion
        chk=$(which chkconfig)
```